test/recipes/03-test_fipsinstall.t

```diff
@@ -12,18 +12,37 @@ use warnings;
 use File::Spec;
 use File::Copy;
 use OpenSSL::Glob;
-use OpenSSL::Test qw/:DEFAULT srctop_file/;
+use OpenSSL::Test qw/:DEFAULT srctop_dir bldtop_dir bldtop_file/;
 use OpenSSL::Test::Utils;
 
-setup("test_fipsinstall");
+BEGIN {
+    setup("test_fipsinstall");
+}
+use lib srctop_dir('Configurations');
+use lib bldtop_dir('.');
+use platform;
 
-plan skip_all => "Test disabled in this configuration"
-    if $^O eq 'MSWin32';
+plan skip_all => "Test only supported in a dso build" if disabled("dso");
```

Comment:
You should check `fips` instead of `dso`...

Reply:
done